src/anybody.js

```diff
@@ -233,12 +232,6 @@ export class Anybody extends EventEmitter {
   }
 
   step() {
-    const { live, withering } = stepLife(this.bodies, this.witheringBodies)
-    this.witheringBodies ||= []
-    this.witheringBodies.push(...withering)
-    this.witheringBodies = stepWithering(this.witheringBodies)
-    this.bodies = live
```

Comment:
ah I see this is removed. Do we want to remove the stepWithering in visuals.js? I'd think we want to remove withering altogether without life.

Reply:
Ja but let's keep the visual part for now since it looks kinda good and we might wanna just start there when adding visuals for reaching `maxStarLvl`?